cmd.go

```diff
@@ -214,13 +218,24 @@ func (r *Runner) runE(runner interface{}, cfg interface{}) func(cmd *cobra.Comma
 
 		l := log.New()
 
+		loglevelFlag, err := cmd.Flags().GetString("loglevel")
+		if err != nil {
+			return err
+		}
+
 		if module, ok := runner.(fx.Option); ok {
 			if _, ok = cfg.(ProvidesFxOptions); ok {
 				runner = func(ctx context.Context, cfg ProvidesFxOptions) error {
```

Comment:
The return value is an error. Would it make sense to store it in a different variable?

Also, I find the usage of module in both line 226 and 239 a little confusing.  Would it make sense to pass the runner in line 239 and ignore the return from type check in line 226?

Reply:
This will eventually get rewritten too. v2.1 :-)